monitoring/workspace-variables/prod.tfvars.json

```diff
@@ -91,7 +87,6 @@
   "internal_apps": [
     "apply-sandbox.apps.internal",
     "apply-prod.apps.internal",
-    "apply-loadtest.apps.internal",
```

Comment:
The jmeter monitors can be removed as well as they are only used for load testing

Reply:
removed jmeter monitors